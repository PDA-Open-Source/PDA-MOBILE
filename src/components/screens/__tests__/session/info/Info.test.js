import React from 'react';
import renderer from 'react-test-renderer';
import {shallow } from 'enzyme';
import Info from '../../../session/info/Info';
jest.mock('react-native-device-info');

describe('Info Page Component',()=>{
  let wrapper ;
  beforeEach(()=>{
    wrapper = shallow(<Info/>);
  });
  describe('Info Page Component rendering',()=>{

    it('Check Info render without break', () =>{
      const tree = renderer.create(<Info/>).toJSON();
      expect(tree).toBeTruthy();
    });

    it('Info Component should render ',()=>{
      const snap = renderer.create(<Info/>);
      expect(snap.toJSON()).toMatchSnapshot();
    });

    it('render scrollView',()=>{
      expect(wrapper.find('ScrollView').length).toBe(1);
    });
  });
});