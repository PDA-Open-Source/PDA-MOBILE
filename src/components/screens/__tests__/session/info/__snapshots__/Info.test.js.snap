// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Page Component Info Page Component rendering Info Component should render  1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        Session Name
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#EEEEEE",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        Session Description
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        Training Org
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#EEEEEE",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        Start Date
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        End Date
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#EEEEEE",
          "flex": 1,
          "flexDirection": "row",
          "paddingHorizontal": 29.426470588235293,
          "paddingVertical": 39.23529411764706,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 2,
            },
          ]
        }
      >
        Location
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Roboto-Regular",
              "fontSize": 28.285714285714285,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {
              "color": "#121212",
              "flex": 3,
            },
          ]
        }
      />
    </View>
  </View>
</RCTScrollView>
`;
